### DISCUSSION TITLE
UUID7 is no longer a draft

### DISCUSSION THREAD

**Discussion by VaZark** [aminalaee/uuid-utils#58] [General]
Looks like  [RFC 9562](https://datatracker.ietf.org/doc/rfc9562/) is no longer a draft and is officially a proposed standard as of May 2024.

Is the library compliant with the latest version? I pose the question as I have been recently burnt by the [uuid7](https://github.com/stevesimmons/uuid7) repository which has been on an older draft for some time now.

**Comment by aminalaee** [comment]
It was already discussed here: https://github.com/aminalaee/uuid-utils/issues/49
The UUID implementation is from the rust UUID, so it's better to double-check there.
